Functions/Misc/colors

```diff
@@ -82,10 +82,13 @@ for k in ${(k)color}; do color[${color[$k]}]=$k; done
 for k in ${color[(I)3?]}; do color[fg-${color[$k]}]=$k; done
 
 # This is inaccurate, but the prompt theme system needs it.
+# Note: "gray" is US spelling.
```

Comment:
I question this comment's cost/benefit ratio.  Suggest to remove it and add the information to the log message instead.

(Not trying to bikeshed; just to keep the comments' signal/noise ratio high.  Cf. https://producingoss.com/en/you-are-what-you-write.html#writing-content.)

Reply:
```suggestion
```